### ISSUE TITLE
Failing to retrieve list of pages

### ISSUE THREAD

**Issue by motlib** [Kenny2github/mw-api-client#7]
When listing the pages of my wiki instance, I get the following error:
```
Traceback (most recent call last):
  File "mw.py", line 34, in <module>
    main()
  File "mw.py", line 29, in main
    for p in pages:
  File "/home/schroe03/tmp/mw/.venv/lib/python3.6/site-packages/mw_api_client/wiki.py", line 79, in _generate
    last_cont[limitkey] = self._wraplimit(params)
  File "/home/schroe03/tmp/mw/.venv/lib/python3.6/site-packages/mw_api_client/wiki.py", line 95, in _wraplimit
    if 'apihighlimits' in getattr(self.currentuser, 'rights', ()):
```

This happens when executing the following code:

```python
import mw_api_client as mw

BOT_USERNAME='...'
BOT_PASSWORD='...'

def main():
    wp = mw.Wiki("https://wiki.cn.kostal.int/wiki/api.php", "PyTestBot/0.0.0")

    # login is only required to 
    r = wp.login(BOT_USERNAME, BOT_PASSWORD)
    if r['result'] == 'Failed':
        print(r)
        return
    pages = wp.allpages(limit='max')
    for p in pages:
        print(p)
        
if __name__ == '__main__':
    main()
```

I use mw-api-client version 3.1.0 with Python 3.6.7 and MediaWiki 1.32.0. 

It seems the access to the rights attribute cannot work, as rights(...) is a method of the currentuser object and not a list of the available access rights.

**Comment by Kenny2github** [comment]
"wiki.cn.kostal.int" is an invalid domain; if I replace `"https://wiki.cn.kostal.int/wiki/api.php"` with `"https://en.wikipedia.org/w/api.php"` it works as expected. Could you provide a more publicly-accessible repro case?

**Comment by motlib** [comment]
First of all: Thanks for answering my issue report so fast! 

Unfortunately I cannot give you access to the wiki, as it is an internal company wiki. But I have tried a few more things:
* Log in with a standard users bot account to my wiki (until now I tried with the bot account of a member of administrator group): This does not change the result, the exception still happens.
* Enumerate the pages in my wiki without logging in: The exception does not occur. 
* Enumerate pages of `https://en.wikipedia.org/w/api.php` without login: The exception does not occur (same result as you have)

So it seems to me, that the exception occurs as soon as you log in before enumerating the pages.
